aicsdeformation/finishers/cell_deformation_overlays.py

```diff
@@ -0,0 +1,76 @@
+import matplotlib.pyplot as plt
+import numpy as np
+from pathlib import Path
+from typing import List
+from skvideo.io import FFmpegWriter
+
+from ..types import Displacement
+from ..loaders.path_images import PathImages
+
+
+class OverlayFinisher:
+    """
+    This class uses the generated images and displacements (u, v) to create deformation heatmaps
+    with cell max-projection overlays
+    """
+    def __init__(self, displacement_list: List[Displacement], cell_images: PathImages, over_path: Path):
+        self.disps = displacement_list
+        self.cell_images = cell_images
+        self.over_home = over_path
+        self.over_images = PathImages()
```

Comment:
I think `over_home` and `over_images` are basically the directory or images to use to act as the images to then overlay heatmaps onto, but there is no parameter documentation here.